pkg/webhook/hook.go

```diff
@@ -304,7 +308,7 @@ func injectNodeSelector(nodeSelector map[string]string, ignoreLabels []string) m
 		nodeSelectorBackup[k] = v
 	}
 	for k, v := range nodeSelector {
-		// not found in label, delete
+		// delete if found in ignoreLabels
```

Comment:
We just want to keep labels in ignoreLabel still in `Affinity` or `NodeSelector`.

So the comment is incorrect.